packages/ingest/granule.js

```diff
@@ -619,6 +629,33 @@ async function moveGranuleFile(source, target, options) {
   return s3.deleteObject(source).promise();
 }
 
+/**
+ * move granule files from one s3 location to another
+ *
+ * @param {Array} sourceFiles - array of file objects
+ * @param {Object} destination - bucket and key defining the destination of the granule
+ * @param {string} destination.bucket - aws bucket
+ * @param {string} destination.key - filepath on the bucket for the destination
+ * @returns {Promise<undefined>} returns `undefined` when all the files are moved
+ */
+async function moveGranuleFiles(sourceFiles, destination) {
+  const moveFileRequests = sourceFiles.map((file) => {
+    const source = {
+      Bucket: file.bucket,
+      Key: file.filepath
```

Comment:
This was `file.filename`, but I changed it to `file.filepath` (was getting `CopySource: 'cumulus-test-sandbox-private/s3://cumulus-test-sandbox-private/MOD09GQ/MOD/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met'`)

Can you write a test for the `moveGranuleFiles` function?